src/main/java/edu/iis/powp/adapter/LinePlotterDrawerAdapter.java

```diff
@@ -11,7 +11,7 @@
     private int startX = 0, startY = 0;
     private ILine line;
 
-    public LinePlotterDrawerAdapter(DrawPanelController drawPanel, ILine basicLine) {
```

Comment:
Actually, injecting ApplicationWithDrawer.getDrawPanelController() to drawPanel would make possible to avoid coupling with ApplicationWithDrawer class.